### PR TITLE
Task_1 completed!

### PR DESCRIPTION
**Pull request by mikita-kandratsyeu**

### CODE REVIEW

File: src/js/task_1/script.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
+/**
+ * Task 1.1
+ * Implementing the following functions without any libs
+ */
+
+function sum(a, b) {
```
Comment (Honneybalma) [inline src/js/task_1/script.js:6]: Try to use arrow functions!

File: src/js/task_1/script.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+console.log(equilateralTriangle(10));
+
+function rhombus(lines) {
+// function should return string
+// (2) =>
+//   *
+//  ***
+// *****
+//  ***
```
Comment (Honneybalma) [inline src/js/task_1/script.js:187]: Don't use comments that don't make any sense!

**Review by Honneybalma — COMMENTED** [review]
Looks good to me!

File: src/js/task_1/script.js (hunk 2 above)
Comment (mikita-kandratsyeu) [reply src/js/task_1/script.js:187]: These comments are just for pretty looks of code!

**Pull request merged**